assets/timber.scss.liquid

```diff
@@ -1767,15 +1822,14 @@ label.error {
   > li {
     position: relative;
     display: inline-block;
-
-    &:first-child a {
-      padding-left: 0;
-    }
   }
+}
 
-  @include at-query ($max, $medium) {
-    margin: 0 auto;
-    text-align: center;
+.site-nav--mobile {
+  @extend .site-nav;
```

Comment:
depending on how many styles `.site-nav` includes, might be better off doubling up the class on the HTML element? eg - `site-nav--mobile site-nav`? Could be an interesting discussion for FED talks